src/nvim/buffer.c

```diff
@@ -1887,7 +1887,7 @@ int ExpandBufnames(char_u *pat, int *num_file, char_u ***file, int options)
         *file = xmalloc(count * sizeof(**file));
```

Comment:
The original patch has another `vim_regfree()` here, for when malloc fails. Since we don't provide for the OOM situation, I suppose it's safe to skip the `vim_regfree`. Correct?